packages/mobile/src/qrcode/QRScanner.tsx

```diff
@@ -61,10 +63,19 @@ class QRScanner extends React.Component<Props, State> {
     navigate(Screens.QRCode)
   }
 
+  componentDidMount() {
+    if (Platform.OS === 'android') {
+      this.props.setRequestingAndroidPermission(true)
```

Comment:
I worry that needing to set this on every screen where we possibly request perms is error prone. Someone will surely forget one eventually. Is this absolutely necessary? If so, can this go into `packages/mobile/src/utils/permissions.android.ts`

Reply:
It is already in `packages/mobile/src/utils/permissions.android.ts`. Here I've added it just because we use `RNCamera`, which does not use `packages/mobile/src/utils/permissions.android.ts` and therefor require a separate implementation.